drawing_2.css

```diff
@@ -0,0 +1,56 @@
+.container{
+	width: 300px;
+	height: 100px;
+}
+.magenta{
+	background-color: #FF00FF;
+	width: 100px;
+	height: 100px;
+	float: right;
```

Comment:
Just make all three `divs` per row `float:left;` keeps it simple and easier to understand.